### PR TITLE
Add new maintainers

### PR DESCRIPTION
**Pull request by psychbot**

### CODE REVIEW

**Review by khushbr — APPROVED** [review]

**Review by getsaurabh02 — APPROVED** [review]

**Comment by bbarani** [comment]
@getsaurabh02 @khushbr hushbr maintainers.md file and codeowners should reflect the same list and none of these new folks are added as maintainers yet. You need to tag @opensearch-project/admins to add them as maintainers before merging this PR. codeowners file should be valid before merging this PR.


Screenshot 2024-02-29 at 12 31 44 PM

**Pull request merged**
